packages/storybook-builder-vite/codegen-iframe-script.ts

```diff
@@ -1,6 +1,7 @@
 import { loadPreviewOrConfigFile } from '@storybook/core-common';
 import { normalizePath } from 'vite';
 import { listStories } from './list-stories';
+import { default as slash } from 'slash';
```

Comment:
Question: why not just `import slash from 'slash'`?

Reply:
Hmm, I was seeing a weird error but I think that was in a different setup. I can just use the normal import